From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Sun, 26 Jul 2020 06:45:17 +1200
Subject: Make tar quiet when building the judgehost docker image

The file listing is very long and bloats the log.
---
 docker/judgehost/chroot-and-tar.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/docker/judgehost/chroot-and-tar.sh b/docker/judgehost/chroot-and-tar.sh
index 4e2b12ea5..6fb676aa0 100755
--- a/docker/judgehost/chroot-and-tar.sh
+++ b/docker/judgehost/chroot-and-tar.sh
@@ -2,5 +2,5 @@
 /opt/domjudge/judgehost/bin/dj_make_chroot
 
 cd /
-tar -czvpf /chroot.tar.gz /chroot
-tar -czvpf /judgehost.tar.gz /opt/domjudge/judgehost
+tar -czpf /chroot.tar.gz /chroot
+tar -czpf /judgehost.tar.gz /opt/domjudge/judgehost
-- 
2.20.1

